src/web/resources/report-page.js

```diff
@@ -390,27 +452,82 @@ function buildBody(jsonData, otherJsonData, filterFunction) {
         ])
     ])
 
+    function buildSortableTextElement(stringName, nestedArray) {
+        const nestedElements = [sortState[stringName] ? `${toTitleCase(stringName)} ` + `${sortDescending ? "⏶" : "⏷"}` : `${toTitleCase(stringName)} –`, nestedArray]
```

Comment:
What is the point of doing `${foo}`? It is literally just `foo`

Reply:
Defensive type casting 🤦‍♂️.